storage_service/locations/models/package.py

```diff
@@ -1021,11 +1027,12 @@ def finish_reingest(self, origin_location, origin_path, reingest_location, reing
         # TODO what do if LOCKSS & uncompressed?
         if self.package_type in (Package.AIP, Package.AIC):
             reingest_pointer_src = os.path.join(origin_location.relative_path, os.path.dirname(origin_path), 'pointer.xml')
-            reingest_pointer_name = 'pointer.' + self.uuid + '.reingest.xml'
-            reingest_pointer_dst = os.path.join(ss_internal.relative_path, reingest_pointer_name)
-            src_space.move_to_storage_service(reingest_pointer_src, reingest_pointer_name, internal_space)
-            internal_space.move_from_storage_service(reingest_pointer_name, reingest_pointer_dst)
-            reingest_pointer = os.path.join(ss_internal.full_path, reingest_pointer_name)
+            if os.path.isfile(os.path.join(origin_location.full_path, os.path.dirname(origin_path), 'pointer.xml')):
```

Comment:
`origin_location.full_path` may not be local to this machine.  For example, if the origin_location's Space is a PipelineFS, only accessible via SSH. This change should probably be removed.